src/server/debug.rs

```diff
@@ -813,6 +819,19 @@ impl<ER: RaftEngine> Debugger<ER> {
             })
     }
 
+    pub fn get_store_info(&self) -> Result<DebugStoreInfo> {
```

Comment:
get_api_version maybe better since we already have the `get_cluster_id` and `get_store_id`.

Reply:
@andylokandy , `get_api_version ` only get api version information, When execute `store` command, we expect output valuef of store_id and api_version, if change to   `get_api_version `, I'am afraid we may call the same method and get data from rocksdb twice

Reply:
Can we refactor all these threes into a single `get_store_indent`?

Reply:
Sounds reasonable. i have merge three into one